url-bar.html

```diff
@@ -42,22 +41,36 @@
         overflow-x: auto;
         overflow-y: hidden;
         background-color: white;
-        @apply(--url-bar);
+        @apply --url-bar;
       }
-      :host[in-iframe] {
+      :host([in-iframe]) {
         /* This element only wants to be displayed if it's in an iframe. */
         display: block;
       }
+
       label {
-        @apply(--paper-font-common-base);
+        font-family: 'Roboto', 'Noto', sans-serif;
+        -webkit-font-smoothing: antialiased;
```

Comment:
I'd apply the mixin right after this line, in case some user relies on it to override fonts